spec.html

```diff
@@ -343,6 +343,11 @@ <h3>Notes</h3>
     <li>Max key - Special type which compares higher than all other possible BSON element values.</li>
     <li>Generic binary subtype - This is the most commonly used binary subtype and should be the 'default' for drivers
       and tools.</li>
+    <li>Compressed BSON Column - This data type permits compact storage
+      by using BSON type-specific delta or delta-of-delta compression
+      stored in Simple-8b blocks, which uses few bits per element. This
+      type also enables efficient encoding for sparse arrays containing
+      missing values.</li>
```

Comment:
Thank you for doing this! I would rephrase it a little bit and focus on capabilities rather than internals. How about this:

```
Compressed BSON Column - Compact storage of BSON data. Uses delta and delta-of-delta compression as well as run-length-encoding for efficient element storage and has an encoding for sparse arrays containing missing values.
```